### PR TITLE
Add German README

### PR DESCRIPTION
**Pull request by 3urobeat**
This PR adds a german README translation and links to it from the other translations.

### CODE REVIEW

File: README_de_DE.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+| Rückblicke (heute vor x Jahren)                      | Ja     | Ja    |
+| Offline Unterstützung                                | Ja     | Nein  |
+| Schreibgeschützte Gallerie                           | Ja     | Ja    |
+| Gestapelte Bilder                                    | Ja     | Ja    |
+
+## Unterstütze das Projekt
+
+Ich habe mich diesem Projekt verpflichtet und werde nicht aufgeben. Ich werde die Dokumentation weiter aktualisieren, neue Funktionen hinzufügen und Fehler beheben. Allerdings kann ich das nicht alleine schaffen. Daher brauche ich eure Unterstützung mir zusätzliche Motivation zu geben um weiterzumachen.
+
+Wie unsere Gastgeber in der [selfhosted.show - In der Episode 'The-organization-must-not-be-name is a Hostile Actor'](https://selfhosted.show/79?t=1418) gesagt haben, ist dies ein riesiges Unterfangen, welchem das Team und ich uns annehmen. Ich würde mich liebend gern diesem Projekt in der Zukunft Vollzeit widmen und bitte daher um eure Unterstützung.
```
Comment (danieldietzler) [inline README_de_DE.md:106]: "Eure" capitalized in both paragraphs.

File: README_de_DE.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+Wenn du denkst dass dies die richtige Sache ist und dich selbst die App für eine längere Zeit nutzen siehst, dann denke bitte darüber nach das Projekt mit einer der unten aufgelisteten Optionen zu unterstützen.
```
Comment (danieldietzler) [inline README_de_DE.md:108]: "du" capitalized as well because we address the reader directly.

Comment (danieldietzler) [reply README_de_DE.md:108]: ", dass"

**Review by danieldietzler — COMMENTED** [review]
Thanks for the translation! Some minor nitpicks ;)

**Comment by 3urobeat** [comment]
Thanks for the feedback, your recommendations sound better than what I had wrote previously! :)  
I addressed them in the commit above.

**Review by danieldietzler — APPROVED** [review]
Thanks a lot! :)

**Pull request merged**